user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/Custom_Operator/CO_Metadata.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: CO_Metadata
+---
+
+# Modifying links to DataMiner objects
+
+In a custom operator, you can link row or modify any existing links to DataMiner objects.
+These metadata links allow you to feed the objects in the Dashboards and Low-Code Apps to other components.
```

Comment:
In Dashboards and Low-Code Apps (drop the "the")

Reply:
Done.